src/tribler/core/utilities/search_utils.py

```diff
@@ -1,13 +1,18 @@
 """
 Search utilities.
 
-Author(s): Jelle Roozenburg, Arno Bakker
+Author(s): Jelle Roozenburg, Arno Bakker, Alexander Kozlovsky
```

Comment:
:)

Reply:
We can remove the line with authors altogether. I added the name just because the original people mentioned here do not have a relation to 85% of the code in this file anymore :)